src/coreComponents/linearAlgebra/utilities/LinearSolverParameters.hpp

```diff
@@ -213,8 +213,10 @@ struct LinearSolverParameters
       compositionalMultiphaseReservoir, ///< reservoir with finite volume compositional multiphase flow
       hydrofracture,                    ///< hydrofracture
       lagrangianContactMechanics,       ///< Lagrangian contact mechanics
-      singlePhasePoroelastic,           ///< single phase poroelastic with finite volume single phase flow
-      hybridSinglePhasePoroelastic      ///< single phase poroelastic with hybrid finite volume single phase flow
+      singlePhaseHybridFVM,             ///< hybrid finite volume single-phase flow
```

Comment:
Maybe we should remove this one for now since this is not implemented in the present PR. I have it in a separate branch, so I can add in another PR later to avoid delaying this one.

Reply:
We should keep it because it's need to select the appropriate strategy for the `SinglePhasePoromechanicsSolver`:

https://github.com/GEOSX/GEOSX/blob/fb01475e68977235d0ba0a4fe53a2e9eb373d4f1/src/coreComponents/physicsSolvers/multiphysics/SinglePhasePoromechanicsSolver.cpp#L135-L141